src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -209,16 +214,62 @@ private void AppendStorageProviderValuesToDebugString(StringBuilder sb)
             sb.Append(" }, ");
         }
 
+        internal void ValidateHubName(string hubName)
```

Comment:
So this logic actually used to be here, but I moved it out. The reason is that this validation is very Azure Storage specific, so I needed to move it into a place where it won't affect other durability providers in the future. I would like to keep this in the `AzureStorageOptions`.